scripts/release/write-changelog.ts

```diff
@@ -77,6 +77,35 @@ const writeToFile = async ({
   await writeFile(changelogPath, nextChangelog);
 };
 
+const writeToDocsVersionFile = async ({
+  changelogText,
+  version,
+  verbose,
+}: {
+  changelogText: string;
+  version: string;
+  verbose?: boolean;
+}) => {
+  const isPrerelease = semver.prerelease(version) !== null;
+  const filename = isPrerelease ? 'next.json' : 'latest.json';
```

Comment:
I assume that you ensure that this never gets called in any other scenarios? Future release? Canary release?

Reply:
That's a good point. For now we only support the two release types, so it's hard to come up with how the logic would be if there were other types of releases here.